conf/eddie.config

```diff
@@ -11,8 +11,8 @@ executor {
 
 process {
     resourceLimits = [
-        memory: 384.GB,
-        cpus: 32,
+        memory: 1024.GB,
+        cpus: 64,
```

Comment:
You should also apply these to the old style `max_*` parameters below (lines 48-50)